install-instructions.php

```diff
@@ -389,6 +389,8 @@
 		      <a href="<?php echo $DOWNLOAD_CLIENT_DESKTOP_TEST_SOURCES; ?>" class="btn btn-lg btn-default"><i class="icon-archive"></i> Sources</a>
 		    </div>
 		    <?php } ?>
+                    <h2>Android Beta Client</h2>
+                    <p>If you want to test the Android Beta Client, you can <a href="https://f-droid.org/packages/com.owncloud.android.beta/">download it on Fdroid.</a></p>
```

Comment:
Same here:

"Our mobile team provides regular beta releases. You can install them from the F-Droid Android app store (link). Please report any issues to the tracker (link)."